frontend/tailwind.config.js

```diff
@@ -8,19 +9,37 @@
 /** @type {import('tailwindcss').Config} */
 const plugin = require('tailwindcss/plugin');
 
+// eslint-disable-next-line no-undef
 module.exports = {
     content: ['./src/**/*.{js,jsx,ts,tsx}'],
     theme: {
         extend: {
             colors: {
-                react: {
-                    DEFAULT: '#282c34',
+                // Color thema of the application
+                // can be used in normal tailwind syntax, e.g.,
+                // bg-primary-dark for to use '#002884'
+                primary: {
+                    light: '#757ce8',
+                    main: '#3f50b5',
+                    dark: '#002884',
+                    contrastText: '#fff',
+                },
+                secondary: {
+                    light: '#ff7961',
+                    main: '#f44336',
+                    dark: '#ba000d',
+                    contrastText: '#000',
+                },
+                error: {
+                    light: '#ef5350',
+                    main: '#d32f2f',
+                    dark: '#c62828',
                 },
 
-                //Custom colors, added as css classes like bg-textfield, font-textfield etc.
-                textfield: '#495E77',
-                textcolor: '#C7D3D1',
-                panel: '#334155',
+                background: '#CFD8DC',
+                surface: '#fff',
+                onSurface: '#000',
```

Comment:
Just use `bg-black` instead of having these separately. These hide the actual color, which takes away some of TW's benefits, that relate to seeing directly from classes that how these look.